pkg/factory/risingwave_object_factory.go

```diff
@@ -180,6 +184,18 @@ func (f *RisingWaveObjectFactory) componentName(component, group string) string
 	}
 }
 
+func (f *RisingWaveObjectFactory) componentAddr(component, group string) string {
+	componentName := f.componentName(component, group)
+	// TODO according to @arkbriar, update compute advertise-addr may cause compatibility issue
+	if component == consts.ComponentCompute {
+		return componentName
+	}
```

Comment:
It's okay since the default value of `isFullKubernetesAddr` is false.

Reply:
Just remove this `if-block` and let the users know that `enableFullKubernetesAddr` will cause incompatibility